### PR TITLE
Let WPML Media translate URLs in the post body

### PR DESCRIPTION
**Pull request by strategio**
A context is added for the filter `wpml_pb_should_body_be_translated` (see https://git.onthegosystems.com/wpml/wpml-media-translation/merge_requests/266).

https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlcore-5923

### CODE REVIEW

**Comment by strategio** [comment]
@OnTheGoSystems/wpml please review.

File: src/class-wpml-gutenberg-integration.php [hunk 1]
```diff
@@ -258,4 +261,27 @@ public function wpml_config_filter( $config_data ) {
...
 	}
 
+	/**
+	 * @param bool    $bool
+	 * @param WP_Post $post
+	 * @param string  $context
+	 *
+	 * @return bool
+	 */
+	public function should_body_be_translated_filter( $bool, WP_Post $post, $context = '' ) {
```
Comment (brucepearson) [inline src/class-wpml-gutenberg-integration.php]: @strategio I would rename `$bool` to `$translate`

File: src/class-wpml-gutenberg-integration.php [hunk 2]
```diff
@@ -258,4 +261,27 @@ public function wpml_config_filter( $config_data ) {
...
+		if ( 'translate_images_in_post_content' === $context && $this->is_gutenberg_post( $post ) ) {
+			return true;
```
Comment (brucepearson) [inline src/class-wpml-gutenberg-integration.php]: @strategio I would change this to `$translate = true;`

**Review by brucepearson — DISMISSED** [review]

File: src/class-wpml-gutenberg-integration.php (hunk 1 above)
Comment (strategio) [reply src/class-wpml-gutenberg-integration.php]: changed @brucepearson (I was not inspired 😀)

**Review by andreasciamanna — APPROVED** [review]

**Pull request merged**